internal/rollup/rollup.config.js

```diff
@@ -0,0 +1,59 @@
+const rollup = require('rollup');
+const nodeResolve = require('rollup-plugin-node-resolve');
+const commonjs = require('rollup-plugin-commonjs');
+const fs = require('fs');
+const path = require('path');
+
+class NormalizePaths {
+  resolveId(importee, importer) {
+    // process.cwd() is the execroot and ends up looking something like /.../2c2a834fcea131eff2d962ffe20e1c87/bazel-sandbox/872535243457386053/execroot/<workspace_name>
+    // from the path to the es6 output is bazel-out/host/bin/<build_file_path>/rollup.runfiles/<workspace_name>/<build_file_path>/bazel-out/host/bin
+    const workspaceName = process.cwd().split('/').pop();
+
+    const firstSegment = importee.split('/')[0]
+    const importerDir = importer ? path.dirname(importer) : ""
+    const buildFileMatch = importerDir.match(`rollup\\.runfiles\\/${workspaceName}/([\\w\\-. \\/]+)\\/bazel\\-out`)
+    const buildFilePath = buildFileMatch ? buildFileMatch[1] : "";
+
+    if (firstSegment === 'bazel-out') {
```

Comment:
the `require` function in nodejs was already patched to allow Bazel loading semantics, so maybe you can replace all this path handling logic with `require.resolve(importee, importer)`

Reply:
Interesting. Will try it out.

Reply:
Doesn't quite work. `require.resolve(importee, importer)` ends up resolving to the es5 source. For example:

```
importee: angular_bazel_example/src/hello-world/hello-world.module
importer: /private/var/tmp/_bazel_greg/2c2a834fcea131eff2d962ffe20e1c87/bazel-sandbox/4434148002827539973/execroot/angular_bazel_example/bazel-out/host/bin/src/rollup.runfiles/angular_bazel_example/src/bazel-out/host/bin/src/./app.ngfactory.js
require.resolve(): /private/var/tmp/_bazel_greg/2c2a834fcea131eff2d962ffe20e1c87/execroot/angular_bazel_example/bazel-out/host/bin/src/hello-world/hello-world.module.js
manual resolve: /private/var/tmp/_bazel_greg/2c2a834fcea131eff2d962ffe20e1c87/bazel-sandbox/4434148002827539973/execroot/angular_bazel_example/bazel-out/host/bin/src/rollup.runfiles/angular_bazel_example/src/bazel-out/host/bin/src/hello-world/hello-world.module.js
```

Reply:
Ok. rollup.config.js is as simple as I think I can make it. Templated and using require.resolve for workspace_name imports.